lib/travis/build/script/node_js.rb

```diff
@@ -222,7 +222,12 @@ def iojs_3_plus?
 
           def npm_install(args)
             sh.fold "install.npm" do
-              sh.cmd "npm install #{args}", retry: true
+              sh.if "$(vers2int `npm -v`) -gt $(vers2int 5.8.0)" do
```

Comment:
It's worth noting that I'm not 100% sure how the `vers2int` command works here, and this _might_ be wrong

Reply:
You'd want `-ge`, not `-gt`.

Reply:
You'd also need additional conditions, as discussed in this PR.